src/main/Config.h

```diff
@@ -248,6 +252,22 @@ class Config : public std::enable_shared_from_this<Config>
     // processes `FLOW_CONTROL_SEND_MORE_BATCH_SIZE` messages
     uint32_t FLOW_CONTROL_SEND_MORE_BATCH_SIZE;
 
+    // A config parameter that controls how many bytes worth of flood messages
+    // (tx or SCP) from a particular peer core can process simultaneously
+    uint32_t PEER_FLOOD_READING_CAPACITY_BYTES;
```

Comment:
I think it will be nice to add a description in this PR to explain peer flood capacity, batch size for both sender and  receiver (maybe a simple graph will be better?)

Reply:
Added!